planner/core/logical_plans.go

```diff
@@ -454,14 +461,17 @@ func (ds *DataSource) deriveTablePathStats(path *accessPath) (bool, error) {
 // deriveIndexPathStats will fulfill the information that the accessPath need.
 // And it will check whether this index is full matched by point query. We will use this check to
 // determine whether we remove other paths or not.
-func (ds *DataSource) deriveIndexPathStats(path *accessPath) (bool, error) {
+func (ds *DataSource) deriveIndexPathStats(path *accessPath, considerConditions []expression.Expression) (bool, error) {
```

Comment:
Add some comments for `considerConditions`.

Reply:
reset the value of pushedDownConds?